workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -98,10 +251,7 @@ public static void Launch()
                 }
                 else
                 {
-                    var content = File.ReadAllText(latestLogFile.FullName);
-                    message = $"{message}\n{content}";
-
-                    Debug.LogError(message);
+                    Debug.LogError($"{message}");
```

Comment:
Looks like string interpolation is no longer necessary :)

Reply:
Or the `if/else` block itself

Reply:
Fixed it now so it makes more sense.